rubocop-mdsol.gemspec

```diff
@@ -1,29 +1,29 @@
 # frozen_string_literal: true
 
+require_relative "lib/rubocop/mdsol/version"
+
 Gem::Specification.new do |spec|
   spec.name     = "rubocop-mdsol"
-  spec.version  = "0.2.0"
+  spec.version  = RuboCop::Mdsol::VERSION
   spec.authors  = ["Team Æ", "Team 10"]
   spec.email    = ["ae@mdsol.com", "team10@mdsol.com"]
   spec.license  = "MIT"
   spec.summary  = "Base RuboCop configuration files for Ruby projects at Medidata"
 
   spec.metadata = {
     "homepage_uri" => "https://github.com/mdsol/rubocop-mdsol",
-    "changelog_uri" => "https://github.com/mdsol/rubocop-mdsol/blob/develop/CHANGELOG.md"
+    "changelog_uri" => "https://github.com/mdsol/rubocop-mdsol/blob/develop/CHANGELOG.md",
+    "rubygems_mfa_required" => "true"
```

Comment:
This was warned and auto-corrected by this cop: https://docs.rubocop.org/rubocop/cops_gemspec.html#gemspecrequiremfa
It sounds useful. But please let me know if it should be removed. 🙇🏻